tenants-certified-attributes-updater/src/main/resources/logback.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration debug="false">
+
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="it.pagopa.interop.commons.logging.LoggerLayout"/>
+        </encoder>
+    </appender>
+
+    <appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
+        <queueSize>8192</queueSize>
+        <neverBlock>true</neverBlock>
+        <appender-ref ref="CONSOLE"/>
+    </appender>
+
+    <logger name="it.pagopa.interop.tenantscertifiedattributesupdater.service" level="ERROR"/>
+    <logger name="it.pagopa.interop.tenantscertifiedattributesupdater" level="INFO"/>
+
+    <root name="application" level="ERROR">
```

Comment:
I don't want to print all the logs from the tenant process client, this is why I change the log level like that, otherwise, we generate a log INFO for each call to the tenant process (~ 22K lines, ok just for the first run).
Just to reduce noise, I don't have any particular reason to keep this configuration except the one I described.

Reply:
makes totally sense 👍